Products/CMFPlone/controlpanel/browser/recyclebin.py

```diff
@@ -0,0 +1,40 @@
+# testcp.py
+from plone.app.registry.browser.controlpanel import ControlPanelFormWrapper
+from plone.app.registry.browser.controlpanel import RegistryEditForm
+from plone.z3cform import layout
+from zope import schema
+from zope.interface import Interface
+
+
+class IRecycleBinControlPanelSettings(Interface):
```

Comment:
should go to `plone.base`

Reply:
done in https://github.com/plone/Products.CMFPlone/pull/4168/commits/2ed81c8fcbf8a02f5533409b82c34aa0a6818835